llvm/test/CodeGen/DirectX/sin.ll

```diff
@@ -21,23 +18,11 @@ entry:
 declare float @llvm.sin.f32(float) #1
```

Comment:
If declare should be removed, why not remove sin.f32 as well?

Reply:
> If declare should be removed, why not remove sin.f32 as well?

Deleted. Thanks!